packages/manager/modules/exchange/src/account/update/update.routing.js

```diff
@@ -0,0 +1,34 @@
+import { unescapeDescription } from '../account.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('exchange.dashboard.account.update', {
+    url: '/update',
+    component: 'exchangeAccountUpdate',
+    params: {
+      emailAccount: null,
+    },
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('emailAccount')
+        .then((emailAccount) => {
+          return !emailAccount ? 'exchange.dashboard.account' : '';
+        }),
+    resolve: {
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('exchange_account_update'),
+
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+
+      emailAccount: /* @ngInject */ ($transition$) => {
+        const { emailAccount } = $transition$.params();
+        return emailAccount
+          ? {
+              ...emailAccount,
+              description: unescapeDescription(emailAccount.description),
+            }
+          : undefined;
```

Comment:
**nitpick**
To explicitly says it is a null value
```suggestion
          : null;
```

Reply:
It's applied